conf/timeseries-filodb-server.conf

```diff
@@ -9,6 +9,7 @@ filodb {
 
   dataset-configs = [
     "conf/timeseries-dev-source.conf"
+    "conf/timeseries-dev-buddy.conf"
```

Comment:
Please comment this before pushing - otherwise it will affect regular dev testing cycles